### PR TITLE
Add test data for no-UMI option

### PR DESCRIPTION
**Pull request by dladd**
Based on a 2k read subset of the [Greiff 2014 250x2 MiSeq presto workflow example](https://presto.readthedocs.io/en/stable/workflows/Greiff2014_Workflow.html)

Tests associated with features added in [bcellmagic pull req 128](https://github.com/nf-core/bcellmagic/pull/128)

### CODE REVIEW

File: testdata-no-umi/Metadata_test-no-umi.tsv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ID	R1	R2	I1	Source	Treatment	Extraction_time	Population
```
Comment (ggabernet) [inline testdata-no-umi/Metadata_test-no-umi.tsv:1]:
```suggestion
ID	R1	R2	Source	Treatment	Extraction_time	Population
```

Comment (ggabernet) [reply testdata-no-umi/Metadata_test-no-umi.tsv:1]: You probably don't need the index reads if you are not using UMIs, right?

**Review by ggabernet — COMMENTED** [review]
Hi @dladd ,

it's looking good, thanks for adding the test data! I just have one comment regarding the metadata sheet :)

File: README.md [hunk 2]
```diff
@@ -14,6 +14,15 @@ This branch contains test data for the [nf-core/bcellmagic](https://github.com/n
...
+* `metadata.tsv` contains the paths to the associated test data.
+* `Greiff2014_Cprimers.fasta` and `Greiff2014_Vprimers.fasta` contain the C and
+  V primers, respectively
+* The `fastq` files are 2k read subsamples of a human BCRseq MiSeq amplicon
+  sequencing data. The I1 / UMI barcode field in the metadata file is left empty
+  in this case.
+
 ### TCR test data
 
 `testdata-tcr` contains the test data needed to run the pipeline for BCRseq data.
```
Comment (ggabernet) [inline README.md:28]:
```suggestion
`testdata-tcr` contains the test data needed to run the pipeline for TCRseq data.
```
Oups, I should also fix this typo!

**Comment by dladd** [comment]
Thanks @ggabernet ! Incorporated now- I didn't realise the SAMPLESHEET_CHECK logic was clever enough to allow for the dropped UMI file field

**Comment by ggabernet** [comment]
Perfect, looks good to me now!

**Review by ggabernet — APPROVED** [review]
LGTM!

**Pull request merged**